DQM/HcalTasks/plugins/QIE11Task.cc

```diff
@@ -18,6 +18,27 @@ QIE11Task::QIE11Task(edm::ParameterSet const& ps)
   _laserType = ps.getUntrackedParameter<int32_t>("laserType", -1);
   _eventType = ps.getUntrackedParameter<int32_t>("eventType", -1);
 }
+//statis
+void QIE11Task::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+
+  //from class inheritance
+  hcaldqm::DQModule::fillPSetDescription(desc);
+
+  desc.addUntracked<std::string>("name", "QIE11Task");
+  desc.addUntracked<int>("debug", 0);
+  desc.addUntracked<int>("runkeyVal", 0);
+  desc.addUntracked<std::string>("runkeyName", "pp_run");
+  desc.addUntracked<edm::InputTag>("tagQIE11", edm::InputTag("hcalDigis"));
+  desc.addUntracked<double>("cut", 20);
+  desc.addUntracked<int>("ped", 4);
+  desc.addUntracked<int>("laserType", -1);
+  desc.addUntracked<int>("eventType", -1);
+  descriptions.add("qie11Task", desc);
+  // or use the following to generate the label from the module's C++ type
+  //descriptions.addWithDefaultLabel(desc);
```

Comment:
I'd suggest to remove these lines if you're not going to use the `addWithDefaultLabel()`
```suggestion
```